README.md

```diff
@@ -44,7 +48,7 @@ The script uses [Git Hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-H
 
 Since we want to add the changes in our `CHANGELOG.md` as well, we have to use [Git's `--amend` feature](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) to change the previous commit and apply the changes from the script, too.
 
-By changing the last commit, we are triggering the `post-commit` hook another time, since the `--ammend` commit is a commit as well. To prevent an infite execution of this hook, we have to check, whether our generated `CHANGELOG.md` have have changed at all. If it is executed the second time, the file should not be different. 
+By changing the last commit, we are triggering the `post-commit` hook another time, since the `--ammend` commit is a commit as well. To prevent an infite execution of this hook, we have to check, whether our generated `CHANGELOG.md` have have changed at all. If it is executed the second time, the file should not be different.
```

Comment:
By changing the last commit, we are triggering the `post-commit` hook another time, since the `--amend` commit is a commit as well. To prevent an infinite loop calling this hook, we check whether our generated `CHANGELOG.md` has changed. The second time the hook is called, the file should not be different.

Reply:
Done